From 810dbc5221a7efc063473da3cca0db5d924b3d4f Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 19 Sep 2016 14:37:09 +0300
Subject: [PATCH 0970/2241] apn806: dts: enable iocc for SD/eMMC interface

Enable IOCC (dma-coherent) for APN806 SD/eMMC interface in DTS

Change-Id: I89f41d5f34d3332690bcbd02e769dbd6a10b6c5f
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32873
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 3042f4d..6984b73f 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -293,6 +293,7 @@
 				clock-names = "core";
 				clocks = <&syscon 4>;
 				xenon,slotno = <0>;
+				dma-coherent;
 				status = "disabled";
 			};
 
-- 
2.7.4

